config/routes.rb

```diff
@@ -7,6 +7,7 @@
       resources :device_registrations, only: %i[create]
       mount_devise_token_auth_for 'User', at: 'auth'
       post '/auth/login', to: 'authentication#login'
+      get '/feed', to: 'feeds#index'
```

Comment:
Use
`resources feeds, only: %i[index]`
instead

Reply:
si hago eso cambio la ruta a feeds y no es la idea

Reply:
Then use
`resource feed, only: %[show]`
And change the controller action to show since it's for a single resource